src/com/rockbite/bootcamp/store/Shop/IShop.java

```diff
@@ -0,0 +1,10 @@
+package com.rockbite.bootcamp.store.Shop;
+
+import com.rockbite.bootcamp.store.IInventory;
+import com.rockbite.bootcamp.store.Product;
+
+public interface IShop {
+    void getProduct(Product it, int count);
```

Comment:
Why do we need a count here in getProduct? getProduct implies we are just returning a product, but this doesn't return anything, and it has parameters that are undefined.